.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
Run whenever we receive a `push` or `pull_request` event
 - https://help.github.com/en/actions/reference/events-that-trigger-workflows#webhook-events